### PR TITLE
feat: Adding API information to A320neo Pilot Briefing

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary
 Adding API information to A320neo Pilot Briefing

This eventually documents every button, switch, dial and knob in the cockpit grouped be panels and interlinked with the interactive flight deck. 

### Location
A320neo Pilot Briefing

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

**Comment by Valastiri** [comment]
suggestion -> for (experimental only) flags on the variables, possibly set a highlight? Makes it a little more visible.

I originally thought maybe put experimental variables into their own separate table but that's a lot of extra work when they roll into other branches.

File: docs/pilots-corner/a32nx-briefing/a32nx_api.md [hunk 1]
```diff
@@ -0,0 +1,973 @@
...
+Flight Deck: [MCDU Panel](flight-deck/pedestal/mcdu.md)
+
+!!! note "The below table shows the API for left MCDU. Replace `L` with `R` for the right MCDU."
+
+| Function    | API Usage               | Values | Read/Write | Type     | Remark |
+|:------------|:------------------------|:-------|:-----------|:---------|:-------|
+| BRT / DIM L | A32NX_MCDU_L_BRIGHTNESS | 0..100 | R/W        | MSFS VAR |        |
+
+
+??? note "MCDU API"
```
Comment (Valastiri) [inline docs/pilots-corner/a32nx-briefing/a32nx_api.md]:
```suggestion
??? note "MCDU API (Click to Expand)"
```
Quick suggestion as this is the only one I could find on this page as an expandable admonition. 

The others are not expandable and users may skip over this without the extra flavor text in the admon.title

**Comment by frankkopp** [comment]
> suggestion -> for (experimental only) flags on the variables, possibly set a highlight? Makes it a little more visible.
> 
> I originally thought maybe put experimental variables into their own separate table but that's a lot of extra work when they roll into other branches.

will - good idea - can't do admonitions in tables but can highlight

**Comment by Rascal650** [comment]
Apologies in advance if it's not the right place to post that.  I started adding the new EFIS events in my mobiflight setup because most of the old EFIS events don't work with the experimental version.   So far so good, I got the Rose, Range and navaid 1 & 2 programmed.   That's easy because they are switches.

The logic for the CSTR/WPT/VOR/etc is a bit different for me. I need to use a "if / then / else" which I've never done before

I need to program the following for each of the 5 buttons when clicked:

  if (A32NX_EFIS_L_OPTION was not set to a different value) then (set A32NX_EFIS_L_OPTION to the current value ) else (set A32NX_EFIS_L_OPTION to 0)

Can someone show me how to do that for the CSTR button?  I can do the other buttons from there. 

THANK YOU!

**Comment by frankkopp** [comment]
> Apologies in advance if it's not the right place to post that. I started adding the new EFIS events in my mobiflight setup because most of the old EFIS events don't work with the experimental version. So far so good, I got the Rose, Range and navaid 1 & 2 programmed. That's easy because they are switches.
> 
> The logic for the CSTR/WPT/VOR/etc is a bit different for me. I need to use a "if / then / else" which I've never done before
> 
> I need to program the following for each of the 5 buttons when clicked:
> 
> if (A32NX_EFIS_L_OPTION was not set to a different value) then (set A32NX_EFIS_L_OPTION to the current value ) else (set A32NX_EFIS_L_OPTION to 0)
> 
> Can someone show me how to do that for the CSTR button? I can do the other buttons from there.
> 
> THANK YOU!

This is not the right place for this question :) - I expected some feedback from you as per DM.

Nonetheless - this is how it looks in SPAD:

image

**Pull request merged**